fedotov_chuvashov/logger.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+#include <string>
+
+namespace uni_course_cpp {
+class Logger {
+ public:
+  void log(const std::string& string);
+  static Logger& get_logger();
+  Logger(const Logger&) = delete;
+  void operator=(const Logger&) = delete;
+
+ private:
+  Logger() {}
```

Comment:
1) Где конструктор, там и деструктор
2) не скобочки, а `default`
```suggestion
  Logger() = default;
  ~Logger() = default;
```